common/lib/xmodule/xmodule/course_module.py

```diff
@@ -834,6 +834,16 @@ class CourseFields(object):
         scope=Scope.settings,
     )
 
+    social_sharing_url = String(
+        display_name=_("Social Media Sharing URL"),
+        help=_(
+            "A fully-qualified URL used by the social sharing icons on the dashboard (if enabled) "
```

Comment:
Thanks Carol -- I've updated the help text for this new field using the
text you've suggested.

Matt

On Fri, Apr 3, 2015 at 4:44 PM, Carol Tong <notifications@github.com> wrote:

> In common/lib/xmodule/xmodule/course_module.py
> <https://github.com/edx/edx-platform/pull/7577#discussion_r27754576>:
>
> > @@ -834,6 +834,16 @@ class CourseFields(object):
> >          scope=Scope.settings,
> >      )
> >
> > +    social_sharing_url = String(
> > +        display_name=_("Social Media Sharing URL"),
> > +        help=_(
> > +            "A fully-qualified URL used by the social sharing icons on the dashboard (if enabled) "
>
> How about:
> If dashboard social sharing and custom course URLs are enabled, you can
> provide a URL (such as the URL to a course About page) that social media
> sites can link to. URLs must be fully qualified. For example:
> http://www.edx.org/course/Introduction-to-MOOCs-ITM001#.
> @marcotuts <https://github.com/marcotuts> @mattdrayer
> <https://github.com/mattdrayer> Can you pls especially comment on whether
> the example URL is an accurate/realistic example? Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/7577/files#r27754576>.
>



-- 
Matthew Drayer
Solutions Architect
edX | MIT
mattdrayer@edx.org